sipcmbeat_ip_anonym.yml

```diff
@@ -0,0 +1,189 @@
+################### Sipcmbeat Configuration Example #########################
```

Comment:
you should probably edit _meta/beat.yml.tmpl and _meta/beat.reference.yml.tmpl  , then run mage update (which should regenerate sipcmbeat.reference.yml and sipcmbeat.yml)